assets/styles/main.css

```diff
@@ -0,0 +1,129 @@
+@font-face {
+    font-family: 'Lucida Grande';
+    src: url(../fonts/lucida-grande/LucidaGrande.ttf);
+}
+
+@font-face {
+    font-family: 'Lucida Grande Bold';
+    src: url(../fonts/lucida-grande/LucidaGrandeBold.ttf);
+}
+
+body{
+    min-width: 900px;
+}
+
+
+.cover{ 
+    min-width: 900px;
+    height: 580px;
+    overflow: hidden;
+    background: url(../imgs/your_verse_hero.jpg) 50% 100% no-repeat;
+    
+}
+
+.cover .title,
+.cover .subtitle,
+.cover .foottitle {
+    display: block;
+    margin-left: auto;
+    margin-right: auto
+}
+
+header{
+    margin: 0 100px;
+}
+
+nav{
+    height: 125px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+}
+
+.navbar{
+    height: 36px;
+    width: 980px;
+    background-image: linear-gradient(rgb(107,107,107) 50%, rgb(122,122,122) 50%);
+    margin: 0 auto;
+    border-radius: 4px;
+    border-style: solid;
+    border-width: .2px;
+    border-color: rgb(80,80,80);
+    display: flex;
+    flex-direction: horizontal;
```

Comment:
`flex-direction` does not accept a `horizontal` as value

Reply:
Done